pinot-core/src/main/java/org/apache/pinot/core/operator/query/AggregationOperator.java

```diff
@@ -63,25 +62,25 @@ protected AggregationResultsBlock getNextBlock() {
     } else {
       aggregationExecutor = new DefaultAggregationExecutor(_aggregationFunctions);
     }
-    TransformBlock transformBlock;
-    while ((transformBlock = _transformOperator.nextBlock()) != null) {
-      _numDocsScanned += transformBlock.getNumDocs();
-      aggregationExecutor.aggregate(transformBlock);
+    ValueBlock valueBlock;
+    while ((valueBlock = _projectOperator.nextBlock()) != null) {
+      _numDocsScanned += valueBlock.getNumDocs();
+      aggregationExecutor.aggregate(valueBlock);
     }
 
     // Build intermediate result block based on aggregation result from the executor
     return new AggregationResultsBlock(_aggregationFunctions, aggregationExecutor.getResult());
   }
 
   @Override
-  public List<Operator> getChildOperators() {
-    return Collections.singletonList(_transformOperator);
+  public List<BaseProjectOperator<?>> getChildOperators() {
```

Comment:
This API needs not to be change right?

Reply:
IDE is showing warning because the generic type is not included